debug_me.py

```diff
@@ -1,3 +1,8 @@
+from collections import namedtuple
+
+Colour = namedtuple('Colour', ['red', 'green', 'blue', 'alpha'])
```

Comment:
```suggestion
Color = namedtuple('Color', ['red', 'green', 'blue', 'alpha'])
```
(personally I don't care, though the whole rest of the code base uses the AE spelling)